js/package.json

```diff
@@ -63,6 +63,7 @@
     "d3-scale": "^2.0.0",
     "d3-svg-legend": "^2.25.6",
     "jquery": "^3.2.1",
+    "lodash": "^4.17.10",
```

Comment:
We are already using underscore which has pretty similar functions to lodash. Perhaps we could look into just using underscore instead of both underscore and lodash.

Reply:
It's needed. They provide different functions.